### ISSUE TITLE
Possible HTTP 409 (Conflict) when searching chat transcripts on startup

### ISSUE THREAD

**Issue by samliew** [samliew/se-electionbot#214]
![Screenshot_2022-11-01_001128](https://user-images.githubusercontent.com/845988/199054474-837e7997-fb63-44ae-885f-4c795aa257f6.png)

```
[chat search] room 249047 (stackoverflow.com)
query: "We have a new nomination Please welcome our latest candidate"

[chat search] room 249047 (stackoverflow.com)
query: "Attention Candidate has withdrawn from the election"

[init] added announced: 0 withdrawn, 0 nominated

[chat search] room 249047 (stackoverflow.com)
query: "moderator election results"

fetch error - https://chat.stackoverflow.com/search?page=1&pagesize=100&room=249047&q=moderator+election+results&user=:
AxiosError: Request failed with status code 409
```

**Comment by samliew** [comment]
Bot crashes with transcript fails to fetch HTTP404

Had to restart twice before it managed to scrape the transcript (which existed)

![Screenshot_2022-11-03_171117](https://user-images.githubusercontent.com/845988/199687035-7da59505-4cf6-4a7c-961a-f1a729be8bb5.png)

```
[transcript] fetching https://chat.stackoverflow.com/transcript/249047/2022/11/3

node:internal/process/promises:279
triggerUncaughtException(err, true /* fromPromise */);
^

HTTPError: Response code 404 (Not Found)
at Request.<anonymous> (/app/node_modules/got/dist/source/as-promise/index.js:118:42)
at processTicksAndRejections (node:internal/process/task_queues:96:5) {
code: 'ERR_NON_2XX_3XX_RESPONSE',
timings: {
start: 1667467632465,
socket: 1667467632465,
lookup: 1667467632466,
connect: 1667467632473,
secureConnect: 1667467632480,
upload: 1667467632480,
response: 1667467632499,
end: 1667467632499,
error: undefined,
abort: undefined,
phases: {
wait: 0,
dns: 1,
tcp: 7,
tls: 7,
request: 0,
firstByte: 19,
download: 0,
total: 34
}
}
}
Process exited with status 1
State changed from starting to crashed
```

**Comment by Oaphi** [comment]
Fixed in cad86c3 and eee26b2
